tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -18,6 +18,7 @@
 from neuroconv import NWBConverter
 from neuroconv.datainterfaces import (
     DeepLabCutInterface,
+    FacemapInterface,
```

Comment:
can you add tests here @alessandratrapani ? we have our test data repository [here](https://gin.g-node.org/CatalystNeuro/behavior_testing_data), can you make a small example file or ask for a small example data that we can add to our testing suite?